### PR TITLE
Release 2.9.0

### PR DESCRIPTION
**Pull request by nicoddemus**
Builds:

* [Travis](https://travis-ci.org/nicoddemus/devpi-cloud-tester/builds/112690759)
* [AppVeyor](https://ci.appveyor.com/project/nicoddemus/devpi-cloud-tester/build/1.0.13)

### CODE REVIEW

File: doc/en/announce/release-2.9.0.rst [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+Happy testing,
+The py.test Development Team
+
+
+2.9.0 (compared to 2.8.7)
+-------------------------
+
+**New Features**
+
+* New ``pytest.mark.skip`` mark, which unconditional skips marked tests.
```
Comment (The-Compiler) [inline doc/en/announce/release-2.9.0.rst]: unconditional -> unconditionally

Comment (nicoddemus) [inline doc/en/announce/release-2.9.0.rst]: Thanks!

**Comment by The-Compiler** [comment]
Thanks - LGTM! :+1:

**Comment by nicoddemus** [comment]
Anyone else have anything to add? Otherwise I plane to finish and publish the release later tonight.

**Pull request merged**